robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -605,3 +624,20 @@ def return_gift_form_titles(self,page=None):
             return section_titles
         else :
             raise Exception ("Invalid argument passed!")
+
+    def verify_progress_bar(self,**kwargs):
+        """Verify batch progress bar and its numbers"""
+        for key,value in kwargs.items():
+            locator=npsp_lex_locators["gift_entry"]["progress_bar"].format(key, value)
+            self.selenium.scroll_element_into_view(locator)
+            self.selenium.wait_until_page_contains_element(locator)
+            
+    def verify_batch_error(self,error,message):
+        """Verify batch error alert present in the page"""
+        locator=npsp_lex_locators["gift_entry"]["batch_error"].format(error,message)
+        self.selenium.scroll_element_into_view(locator)
+        self.selenium.wait_until_page_contains_element(locator)
```

Comment:
these 2 lines should be reverse. You should first wait for element and then scroll to it

Reply:
Done.